src/main/java/com/google/api/codegen/config/DiscoGapicInterfaceContext.java

```diff
@@ -177,7 +179,7 @@ public DiscoGapicInterfaceContext withNewTypeTable(String packageName) {
 
   @Override
   /* Returns a list of public methods, configured by FeatureConfig. Memoize the result. */
-  public Iterable<DiscoveryMethodModel> getPublicMethods() {
+  public List<DiscoveryMethodModel> getPublicMethods() {
```

Comment:
Question: what's the motivation of changing all the `Interable`s to `List`s?

Reply:
We can stream over Lists but not Iterables. 

idk why the Iterable interface had been preferred in the first place? esp when the concrete class is of List type anyway

Reply:
Cool, we would like to promise that order is maintained. Having a deterministic order means the baseline diffs are smaller, and when yoshi regenerates clients, that they don't get meaningless diffs where things are just rearranged.

Reply:
Also, I think you can do `Streams.stream(iterable)` to get a `stream` from an `iterable`.